docker/dev/Dockerfile

```diff
@@ -121,6 +121,7 @@ RUN cd $GOPATH/src/github.com/ligato/vpp-agent \
 
 WORKDIR /
 
+RUN rm -f /dev/shm/db /dev/shm/global_vm /dev/shm/vpe-api
```

Comment:
Executing this command as `RUN` instruction will render its entire purpose useless, because `RUN` instructions are only executed when building an image and these files do not exist at that point yet.

The purpose of this command is to forcefully remove files that might remain there after potential VPP crash, in such situation these files are not cleaned up by VPP, which are used for communication via shared memory (binary API) and we need to remove them before starting VPP again, otherwise connection will fail.

By putting this command in the `RUN` instruction, the files will not be removed if stopped container is started again after VPP crash.

Reply:
Thanks, I better understand th purpose of those command now

Reply:
@ondrej-fabry How exactly are those /dev/shm/... files getting into the container to require cleanup?  If vpp crashes, the whole container goes down, and gets restarted from a fresh clean image.  /dev/shm lives inside the container image, so it goes away with the container.